src/core/dbus-cgroup.c

```diff
@@ -826,6 +829,31 @@ int bus_cgroup_set_property(
 
                 return 1;
 
+        } else if (STR_IN_SET(name, "MemoryLow", "MemoryHigh", "MemoryMax")) {
```

Comment:
> As for warning messages, I'm not sure what to do as it can add up to a lot of noise about various situations

Good point. Maybe, we should `log_debug` as much as possible. And add a one-time warning which warns about using mismatching version.